Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN bundle install
 # Python dependencies for pywb
 WORKDIR /home/was/swap/current/pywb
 RUN pip3 install poetry
-RUN poetry install
+RUN poetry install --no-root
```

Comment:
It looks harmless enough, but can you say what this adds (for my own learning) and how it relates to the pywb bump? An opportunistic change?

Reply:
I'm glad you asked. I only noticed that the latest version of poetry warned about how installing without package was deprecated and that you should add `--no-root` when doing that. So I did that and moved on without really understanding.

Basically I think this means that when doing a `poetry install` poetry looks at the project name `was-pywb` and assumes that there's a`was_pywb` sub-directory (module) to install.  It used to silently pass this by, but it sounds like this will become an error in the next version of poetry.

But reading into it I now see over in https://python-poetry.org/docs/basic-usage/#operating-modes that since we are just using poetry for managing dependencies in the virtual environment we could also add:

```yaml
[tool.poetry]
package-mode = false
```

Which might be a bit clearer?

Reply:
Now that I think about it, this would have failed in the deploy as well, which I didn't update. Changing it in the pyproject.yaml should cover both installs hopefully.

Reply:
The PR has been updated to use the `pyproject.yaml` metadata to cover this instead of relying for the right install command. Thanks for asking about this!